maas/resource_maas_network_interface_bridge.go

```diff
@@ -0,0 +1,262 @@
+package maas
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/maas/gomaasclient/client"
+	"github.com/maas/gomaasclient/entity"
+)
+
+func resourceMaasNetworkInterfaceBridge() *schema.Resource {
+	return &schema.Resource{
+		Description:   "Provides a resource to manage MAAS network Bridges.",
+		CreateContext: resourceMaasNetworkInterfaceBridgeCreate,
+		ReadContext:   resourceMaasNetworkInterfaceBridgeRead,
+		UpdateContext: resourceMaasNetworkInterfaceBridgeUpdate,
+		DeleteContext: resourceMaasNetworkInterfaceBridgeDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceMaasNetworkInterfaceBridgeImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"accept_ra": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Accept router advertisements. (IPv6 only).",
+			},
+			"bridge_fd": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				Description: "Set bridge forward delay to time seconds. (Default: 15).",
+			},
+			"bridge_stp": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				// Computed:    true,
+				Description: "Turn spanning tree protocol on or off. (Default: False).",
+			},
+			"bridge_type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The type of bridge to create. Possible values are: ``standard``, ``ovs``.",
+			},
+			"mac_address": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "MAC address of the interface.",
+			},
+			"machine": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "List of MAAS machines' identifiers (system ID, hostname, or FQDN) that will be tagged with the new tag.",
+			},
+			"mtu": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				Description: "Maximum transmission unit.",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Name of the interface.",
+			},
+			"parent": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Parent interface name for this bridge interface.",
+			},
+			"tags": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Description: "Tags for the interface.",
+			},
+			"vlan": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "VLAN the interface is connected to.",
+			},
+		},
+	}
+}
+
+func resourceMaasNetworkInterfaceBridgeCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
Same as bond, please remove the `Maas` part of every function except `resourceMaasNetworkInterfaceBridge`